### PR TITLE
Create Encounter page

### PR DESCRIPTION
**Pull request by rushaan01**
image

-Page is responsive
-added render test
-API calls yet to be implemented
-added route

### CODE REVIEW

File: forgettable-frontend/src/pages/encounters/CreateEncounter.module.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+@import '../../colours.css';
+
+.Card{
+    display:flex;
+    flex-direction: column;
+    width:910px;
+    height:875px;
+    margin: auto;
+    margin-top: 50px;
+}
```
Comment (noahjade) [inline forgettable-frontend/src/pages/encounters/CreateEncounter.module.css:10]: Can we use the saved var values in the index css? like var(--text-xxlarge) etc ?

**Review by noahjade — COMMENTED** [review]
This is looking really good! Can we change the hard coded fontsize and fontweight to be the css variables? Would also be cool to do this for the colours too, since we want to be able to implement dark theme. You can look in the nav bar component if u need an example of how to get the css vars into the javascript :)

File: forgettable-frontend/src/pages/encounters/CreateEncounter.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import {Autocomplete} from '@mui/material';
+
+
+export default function Encounters() {
+  const options = ['Flynn', 'Jared'];
+  const handleCancelClick = (event) => {
+    alert('canceled');
+  };
+  const handleSaveClick = (event) => {
+    alert('Saved');
```
Comment (rainasong) [inline forgettable-frontend/src/pages/encounters/CreateEncounter.js:16]: Should there be an API call to post the new encounter when Save is clicked?

**Review by rainasong — CHANGES_REQUESTED** [review]

**Review by noahjade — APPROVED** [review]
Thanks for your hard work rushaan!! Lets get this merged so people can help out on the API call stuff :)

**Review by rainasong — APPROVED** [review]

**Review by noahjade — CHANGES_REQUESTED** [review]
Just realised this cant actually be merged!!

**Pull request closed without merging**